building/markdown/style-guide.md

```diff
@@ -66,7 +66,9 @@ Some abbreviations are considered common, useful, and non-technical enough that
 - `etc.` or `etc`
 - `docs`
 
-Contractions (e.g. "won't", "I'm", "that's") should be used sparingly if at all in exercise descriptions, but are not restricted in other language around the site (e.g. website copy, mentoring).
+Contractions (e.g., "won't", "I'm", "that's") should be used sparingly if at all in exercise descriptions, but are not restricted in other language around the site (e.g., website copy, mentoring).
+
+The abbreviations "i.e." and "e.g." should be followed by a comma in American English (see, e.g., this [StackExchange thread](https://english.stackexchange.com/questions/7946/use-of-e-g-are-parentheses-necessary/93658#93658)).
```

Comment:
```suggestion
While many American English style guides state that the abbreviations "i.e." and "e.g." should be followed by a comma in American English (see, e.g., this [StackExchange thread](https://english.stackexchange.com/questions/7946/use-of-e-g-are-parentheses-necessary/93658#93658)), this is not required in Exercism texts.
```

Reply:
I wanted to add "as per overrule by the British BDFL" but that goes a bit far, I think :P